torch/_torch_docs.py

```diff
@@ -3587,6 +3587,54 @@ def merge_dicts(*dicts):
     array([-1,  2,  3])
 """)
 
+add_docstr(torch.frombuffer,
+           r"""
+frombuffer(buffer, dtype=None, count=-1, offset=0, *, requires_grad=False) -> Tensor
+
+Create a 1-dimensional :class:`Tensor` from an object that implements
+the Python buffer protocol.
+
+Interpret :attr:`buffer` as a 1-dimensional buffer of type :attr:`dtype`
+with :attr:`count` elements after the first :attr:`offset` bytes.
+
+The returned tensor and buffer share the same memory. Modifications to
+the tensor will be reflected in the buffer and vice versa. The returned
+tensor is not resizable.
+
+.. note::
+    This function increments the reference count for the object that
+    owns the shared memory. Therefore, such memory will not be deallocated
+    before the returned tensor goes out of scope.
+
+.. warning::
+    The buffer protocols has no notion of devices. Thus, this function only
+    works on CPU buffers. Passing CUDA buffers will probably lead to
```

Comment:
What about this:

"This function's behavior is undefined when passed an object implementing the buffer protocol whose data is not on the CPU. Doing so is likely to cause a segmentation fault." 

?

Reply:
That's way clearer! Thanks for the suggestion.